### PR TITLE
Check DHCP servers are routed after link flap in test_dhcp_relay

### PR DESCRIPTION
**Pull request by bingwang-ms**
Signed-off-by: bingwang <bingwang@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)
This PR is to fix ```test_dhcp_relay_after_link_flap``` failure.
```
 "FAIL: dhcp_relay_test.DHCPTest", 
        "----------------------------------------------------------------------", 
        "Traceback (most recent call last):", 
        "  File \"ptftests/dhcp_relay_test.py\", line 563, in runTest", 
        "    self.verify_relayed_discover()", 
        "  File \"ptftests/dhcp_relay_test.py\", line 445, in verify_relayed_discover", 
        "    \"Failed: Discover count of %d != %d\" % (discover_count, num_expected_packets))", 
        "AssertionError: Failed: Discover count of 0 != 48", 
```
The root cause for the failure is the default route is not rebuilt after uplink flap when test was running.
This PR addressed the issue by adding a ```wait_until``` after link flap to wait until routes to DHCP server are rebuilt.
### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
This PR is to fix ```test_dhcp_relay_after_link_flap``` failure.

#### How did you do it?
This PR addressed the issue by adding a ```wait_until``` after link flap to wait until routes to DHCP server are rebuilt.

#### How did you verify/test it?
Verified on SN4600. All test cases in ```test_dhcp_relay``` pass now.

#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
No.

### CODE REVIEW

File: tests/dhcp_relay/test_dhcp_relay.py [hunk 1]
```diff
@@ -239,9 +250,10 @@ def test_dhcp_relay_after_link_flap(ptfhost, dut_dhcp_relay_data, validate_dut_r
         # Bring all uplink interfaces back up
         for iface in dhcp_relay['uplink_interfaces']:
             duthost.shell('ifconfig {} up'.format(iface))
-
-        pytest_assert(wait_until(50, 5, check_link_status, duthost, dhcp_relay['uplink_interfaces'], "up"),
-                      "Not all uplinks are up")
+
```
Comment (tahmed-dev) [inline tests/dhcp_relay/test_dhcp_relay.py:253]: is the issue happening if the portchannel consists of one interface and doesn't happen otherwise?

**Review by tahmed-dev — APPROVED** [review]
Thanks!

**Review by trzhang-msft — APPROVED** [review]

File: tests/dhcp_relay/test_dhcp_relay.py (hunk 1 above)
Comment (bingwang-ms) [reply tests/dhcp_relay/test_dhcp_relay.py:253]: No, I don't think so. There are two members in each portchannel on this testbed
```
  No.  Team Dev         Protocol     Ports
-----  ---------------  -----------  ---------------------------
 0001  PortChannel0001  LACP(A)(Up)  Ethernet0(S) Ethernet4(S)
 0002  PortChannel0002  LACP(A)(Up)  Ethernet20(S) Ethernet16(S)
 0003  PortChannel0003  LACP(A)(Up)  Ethernet68(S) Ethernet64(S)
 0004  PortChannel0004  LACP(A)(Up)  Ethernet84(S) Ethernet80(S)
```

**Pull request merged**